### ISSUE TITLE
Conformance Requirements NOTE: section question

### ISSUE THREAD

**Issue by airbr** [w3c/sustyweb#23]
https://w3c.github.io/sustyweb/#conformance-requirements

I came to read the guidelines with a question in my head:

> Can I publish a claim that I conform with one of the guidelines, or do I need to comply/conform with all of them?

I found the following Note section not as helpful I'd think possible:

> Although total conformance can technically be achieved by meeting every guideline within the specification, not every website or product, as a general policy, will likely be able to satisfy all Success Criteria. In such situations, it is not recommended that authors prioritize conformance over other important website features.

I understand the need to cover several bases but this seems difficult to read to me. Can total conformance be achieved? Or is it only "technically" achieved?. Wouldn't it be better to say "Some websites will not satisfy all Success Criteria. When not all Success Criteria are met, it is not recommended to prioritize conformance over other important website features".

What are "other important website features"? This seems to imply that important website features would have difficulty conforming. Could this instead say "it is not recommended to prioritize conformance when a major technical barrier has been identified in conforming."?

RE 1.3.2:

> Conformance: A concise description of Sustainability commitments and list of the guidelines adhered to.

Could this concise description include a single guideline that is documented as adhered to? Would that still be a valid conformance claim, albeit with a single guideline?

In summary, I recommend a rewording to something similar to:

>Total conformance is achieved by meeting every Success Criteria for every guideline within the specification. As a general policy most websites or products will not likely be able to satisfy all Success Criteria. When not all Success Criteria are satisfied it is not recommended to prioritize conformance over other important features when a major technical barrier has been identified in conforming.

It is not shorter, just food for thought, and my first attempt at drafting something! Thank you for your time and consideration

**Comment by AlexDawsonUK** [comment]
In general I don't think anyone should put too much effort into thinking about conformance to be honest. We've already spoken about how we're not a checkbox exercise and the spec explicitly declares that the document is just a jumping off point to get people thinking about web sustainability.

It's also critically important to note that if you don't meet the success criteria for that guideline then you have not conformed to that guideline. That could be something if not noted elsewhere I'll make sure to reemphasize within the document.

Regarding the section: While in theory you could run through the list and potentially achieve everything in the guidelines (total conformance), not every website will be able to meet them (whether through the time commitment involved - say the amount of refactoring, if say you're just a solo dev or a hobbyist for example; or as another example there may be situations where some guidelines may simply not apply - and common sense judgements should overrule just trying to conform for the sake of conformance). Every business is going to have to prioritize sustainability as apart of their workflow but if running through this list is going to get in the way of developing important "other features" of their product such as security updates or requested additions, you shouldn't just abandon your current schedule to entirely focus on this (after all, we're pragmatists and realists). I don't think it's necessary to add explicit examples within the spec because as I noted, we're trying to focus on progress over perfection and placing emphasis on conformance, gold stars or grading is a distraction.

Regarding Section 1.3.2: It states in the third bullet you can list specific guidelines you are conforming too if you wish, however I would **strongly** discourage using this as some sort of self-awarding technique. It just might be helpful for the sake of a sustainability statement or a method of proving your meeting of sustainability reduction targets (for internal scope accounting for example).

Though if others have thoughts on the subject feel feee to add comments below!

**Comment by AlexDawsonUK** [comment]
@airbr Based on a re-reading of the conformance section, I'll do the following:
- Declare that all success criteria must be met for a guideline to be considered conformed too.
- Include a clarification for "other website features" to avoid confusion with barriers and conformance.

**Note:** No further clarity at this stage seems necessary regarding claims of conformance (continuing from my points in the previous post). It's reasonably easy for third parties to identify people exploiting the good will of the system, and there are potential legal implications for greenwashing attempts (in multiple nation states).

**Comment by airbr** [comment]
@AlexDawsonUK Thank you for your answers and consideration. Im learning a lot about the project.

**Comment by AlexDawsonUK** [comment]
The conformance section has now been updated to reflect the above, it will appear in the next draft.
